src/commons/utils/corio_utils.py

```diff
@@ -265,6 +275,39 @@ def setup_environment():
     os.makedirs(DATA_DIR_PATH, exist_ok=True)
 
 
+async def activate_degraded_mode() -> None:
+    """Activate degraded mode for execution by shutdown/deleting pod."""
+    bucket = f'object-op-{time.perf_counter_ns()}'.lower()
+    access_key = literal_eval(os.getenv("options")).get("access_key")
```

Comment:
`os.getenv("access_key")` is not enough? What is the need of `literal_eval`

Reply:
os.getenv("access_key") is not enough as same is not stored directly,  I am fetching from environment variables another variable which is dictionary and in environment variables  are stored/fetched as  type string only, There is a dictionary encapsulated in string hence need literal_eval to get real data type.